src/components/NavigationBar/NavigationBar.svelte

```diff
@@ -73,6 +73,11 @@
 
 		const currentFromTop = window.scrollY;
 		showHeader = currentFromTop <= previousFromTop;
+		if (showHeader) {
+			document.documentElement.classList.add('navbar-shown');
```

Comment:
Not a clean way to do it but let's keep it for now. After the merge, I'll create a new issue to refactor this (thinking about Svelte's store) but not a priority.

Reply:
yea, i think it's is better to reference with store in conjunction tailwinds (`class:xxx={$navbarVisibleStore}`)